javascript/javascript1/week1/goodboy-oldboy.js

```diff
@@ -0,0 +1,10 @@
+const dogYearOfBirth = 2021;
+let dogYearFuture = 2045;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let dogYearOrHumanYear = "dog";
+let resultOrInDogYear = true;
+if (!resultOrInDogYear) {
+    dogYear *= 7;
```

Comment:
nice way of re-assigning of variable and the conditional statement 👍

Reply:
thank you for your valuable comments :)